### DISCUSSION TITLE
Canvas not listed in supported LMS Integrations

### DISCUSSION THREAD

**Discussion by tullioa** [community/community#31517] [GitHub Education]
I am trying to integrate my LMS, Canvas, with Github Classroom.  According to[ these instructions](https://docs.github.com/en/education/manage-coursework-with-github-classroom/teach-with-github-classroom/connect-a-learning-management-system-to-github-classroom#generating-configuration-credentials-for-your-classroom), I should be able to find Canvas in the list of LMSs.  However,  only Google Classroom is listed.  How do I get a key and secret for Canvas?

**Comment by blakeCS** [comment]
I was having this issue as well, however it looks like GitHub has turned off the ability to work with Canvas for now. The previous version (1.1?) of integration did not meet the new security requirements, so they are in the process of releasing a new version. The only information that I can find about it says that it will happen "soon". Let me know if you find anything else out!
